### ISSUE TITLE
Cannot install python 3.6 in mac using conda

### ISSUE THREAD

**Issue by ZisenShao** [openai/spinningup#415]
In the installation page, it says I need to create python 3.6 environment using `conda create -n spinningup python=3.6`, but I couldn't find a way to install python 3.6 with conda, both defaults and conda-forge channel don't have this version. Could I use another python version instead? Or may I not use conda to create the environment?

**Comment by ZisenShao** [comment]
I think the main problem is that I am using Mac M2, but the Anaconda version used in spinning up documentation is only for Mac x64, and the arm version of Anaconda does not support Python 3.6. So is there a way to use Mac arm for spinning up?

**Comment by fionalluo** [comment]
Hi, did you end up figuring this out?

**Comment by ZisenShao** [comment]
I ended up using other resources for learning RL, highly recommend UCB CS285, which is open source for homework and lecture. Spinning-up could still be used for reference of papers and algorithms.

**Comment by fionalluo** [comment]
Oh, thank you for the suggestion! I ended up using various hacky solutions to set up spinningup, but the solutions didn't work reliably so I have also switched to other tutorials at this point. Spinning Up's code is pretty out of date I guess

**Comment by Abhijais4896** [comment]
You're right , Python 3.6 is now deprecated and no longer available in many Conda channels by default. In most cases, you can safely use Python 3.7 or 3.8 instead — the codebase should still work as long as the key dependencies are compatible.

You can create the environment like this: 
--> **conda create -n spinningup python=3.8**

if you're not using Conda, you can use **virtualenv** or **venv with Python 3.8+** and manually install the requirements from **requirements.txt**.
